doc/api/addons.markdown

```diff
@@ -1,39 +1,47 @@
 # Addons
 
-Addons are dynamically-linked shared objects. They can provide glue to C and
-C++ libraries. The API (at the moment) is rather complex, involving
-knowledge of several libraries:
+Node.js Addons are dynamically-linked shared objects, written in C or C++, that
+can be loaded into Node.js using the `require()` statement, and used just as if
+they were an ordinary Node.js module. They are used primarily to provide an
+interface between JavaScript running in Node.js and C/C++ libraries.
 
- - V8 JavaScript, a C++ library. Used for interfacing with JavaScript:
-   creating objects, calling functions, etc.  Documented mostly in the
+At the moment, The method for implementing Addons is rather complicated,
+involving knowledge of several components and APIs :
+
+ - V8 JavaScript: the C++ library Node.js currently uses to provide the
```

Comment:
can we `s/V8 JavaScript/V8` to avoid potential confusion about _what_ it is you have to know here?